### DISCUSSION TITLE
Global shortcut for Copy next and previous item

### DISCUSSION THREAD

**Discussion by francwalter** [hluk/CopyQ#2305] [Q&A]
Hi 
how can I add a global shortcut to paste the next item in my list of clips without opening CopyQ?
And the previous item?
E.g. I have in my list:

1. Today
2. is a very
3. nice day

I am writing in my document (e.g. Words or LibreOffice etc.) and want to add one clip after the other with some shortcuts to build:

`<ctrl+v>Today<global-shortcut-copy-next> is a very<global-shortcut-copy-next> nice day`

That possible?
I tried with a little command:

```
copyq:
selectItems(currentItem() + 1)
paste()
```

but that does not work. Neither the inbuilt commands "Paste and copy next".

At the moment I have to open CopyQ, move the selection to the next item, close CopyQ, paste, open CopyQ again and so on.
Could be much faster.
I had this feature in Window's ClipMate (a since long deprecated but a genious and very productive app) where I could fill with few clicks and even fewer mistakes long lists of entries. Very useful then.

Thank.frank

**Comment by reinbeau** [comment]
You want Power Paste.  I am also a longtime Clipmate user (over a decade).  Power Paste stopped working for me some time ago - yes, it would be a useful addition.

**Comment by francwalter** [comment]
Yes, exactly 😄 PowerPaste from ClipMate.

Since there never were any suggestion here, I use since a while two shortcuts, first I go down the list with one shortcut (Copy next item) and then I copy the item. So not that fast and productive as ClipMate and I can make more errors, but it works.

My shortcut for "Copy next item" is Alt+A and "Copy preious item" is Alt+S, exactly what I used in ClipMate 😏 

By the way, did you know that [ClipMate is free](http://clipmate.com/free.htm) now?
Unfortunately it never gots UTF8 support, thats why I quit using it.

**Reply by reinbeau** [reply]
Thank you so much for the link.  I'm not sure what UTF8 support is, would have have been someone picking it up and keeping it developed?  I'll give CopyQ a try.

**Reply by francwalter** [reply]
No, I think its really over.
I asked C. Thornton about the code but no answer.

UTF8 was no issue at the beginning of ClipMate but later it was. German Umlauts e.g.
